cfpb/consumerfinance.gov
# Prepaid search: change agreement bulk_download_path to match bulk.zip

Change underscores in `bulk_download_path` to space characters to match path in bulk.zip
This character change to `path` will not be made if the product name already contains an underscore

## Changes

- Paths that used to contain underscores will now be replaced by the space character as show below
e.g. 
`ABNB Federal Credit Union/CU_Money_VISA_Everyday_Spend_Card/20130801`
will become
`ABNB Federal Credit Union/CU Money VISA Everyday Spend Card/20130801`

## How to test this PR

1. git checkout fix-3913
1. cd cfgov
1. ./manage.py runscript import_prepaid_agreements_data
1. ./manage.py runscript write_prepaid_agreements_data_to_csv
1. Confirm that bulk_download_path in CSV files have space characters as opposed to underscores
1. Download latest bulk.zip file and prepaid CSV files from:
https://www.consumerfinance.gov/data-research/prepaid-accounts/download-agreements/
1. unzip -l bulk.zip
1. grep "CU Money VISA Everyday Spend Card" prepaid_metadata_all_agreements.csv
1. Verify that only path has space characters and ZIP filename in direct_download still has underscores

## Notes
For the following product names that contain an underscore `path` will NOT be changed to an underscore:

- Veracity_Peoples Inc_Espree Prepaid
- Veracity_Community Bank_Espree Prepaid
- PSKW_Nucala_Loyalty Rewards Card
- PSKW_Artemis_Loyalty Rewards Card
- PSKW_Orencia-Loyalty Rewards Card
- Stokes Farm_Harvester Gear-Espree Prepaid MasterCard
- 1st Quality and FNB Employees_Espree Prepaid
- FINAL_VentivaVisaPrepaidDebitCard_04272019
- FINAL_WorldBancardPrepaidMastercard_04272019
- Univision Mastercard Prepaid Card_Direct
- RPM PAYROLL 006_033_542

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
